src/test/java/core/basesyntax/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @Test
+    void reportService_emptyData_Ok() {
+        List<String> data = new ArrayList<>();
+        List<String> expectedReport = new ArrayList<>();
+        expectedReport.add("fruit,quantity");
+        List<String> actualReport = reportService.report(data);
```

Comment:
Create variables closer to their first occurrence.

```suggestion
        List<String> expectedReport = new ArrayList<>();
        expectedReport.add("fruit,quantity");
        List<String> data = new ArrayList<>();
        List<String> actualReport = reportService.report(data);
```